src/test/scala/fr/polytechnique/cmap/cnam/study/pioglitazone/outcomes/BroadBladderCancerSuite.scala

```diff
@@ -177,9 +132,9 @@ class BroadBladderCancerSuite extends SharedContext {
     import fr.polytechnique.cmap.cnam.util.functions.makeTS
 
     val inputDS = Seq(
-      Event[Diagnosis]("Patient1", "main_diagnosis", "g1", "C67", 1.0, makeTS(2006, 1, 20), Some(makeTS(2006, 1, 20))),
-      Event[Diagnosis]("Patient2", "main_diagnosis", "g2", "C67", 1.0, makeTS(2006, 2, 20), Some(makeTS(2006, 2, 20))),
-      Event[Diagnosis]("Patient3", "linked_diagnosis", "g3", "C67", 1.0, makeTS(2006, 3, 20), Some(makeTS(2006, 3, 20)))
+      Event[Diagnosis]("Patient1", "mco_main_diagnosis", "g1", "C67", 1.0, makeTS(2006, 1, 20), Some(makeTS(2006, 1, 20))),
```

Comment:
I think it would be better to use the Category instead of the string. Could you do that please for this suie?

Reply:
Not sure to understand what you mean by that. 
Besides the refacto of the categories I didn't change anything on this test written for the study pioglitazone.

Reply:
Let's leave it like that.